apps/passport-server/src/routing/routes/telegramRoutes.ts

```diff
@@ -51,7 +52,10 @@ export function initTelegramRoutes(
         logger(
           `[TELEGRAM] Redirecting to telegram for user id  ${telegram_user_id}`
         );
-        res.redirect(await telegramService.getBotURL());
+        // TODO: Think about when we want to redirect back to web view or t.me link
+        // res.redirect(await telegramService.getBotURL());
```

Comment:
nice 👍

the redirect doesn't work in the TG webview right? in that case, we can probably just delete this TODO

Reply:
Closed by 5c7e4e07